src/common/delivery/goals/Goal.ts

```diff
@@ -148,16 +152,16 @@ export function hasPreconditions(goal: Goal): goal is GoalWithPrecondition {
 function checkPreconditionStatus(sub: GitHubStatusAndFriends, pg: Goal): { wait?: string, error?: string } {
     const detectedStatus = sub.siblings.find(gs => gs.context === pg.context);
     if (!detectedStatus) {
-        return {wait: "Did not find a status for " + pg.context};
+        return { wait: "Did not find a status for " + pg.context };
     }
     if (detectedStatus.state === "pending") {
-        return {wait: "Precondition '" + pg.name + "' not yet successful"};
```

Comment:
We need to get our IDE config similar so that this kind of pointless change doesn't happen. Maybe tslint rules could fix it: We could investigate the whitespace rules

Reply:
That's probably me. My setup automatically runs `tsfmt --replace`, which is likely the origin of this change. Maybe we just run both `tslint --fix` and `tsfmt --replace`?

Reply:
Can we change the scripts so that it always does both everywhere? I'd be fine with that. I actually prefer the whitespace as it seems more consistent